ansible/roles/condor_configuration/tasks/main.yml

```diff
@@ -1,12 +1,18 @@
 ---
-# - name: template the condor_config.local file
-#   ansible.builtin.template:
-#     src: condor_config.local.j2
-#     dest: /etc/condor/condor_config.local
-#     mode: '0644'
+- name: template the 02-partitionable-slots.config file
+  ansible.builtin.template:
+    src: "02-partitionable-slots.config.j2"
+    dest: "/etc/condor/config.d/02-partitionable-slots.config"
+    mode: "0644"
+
+- name: template the 20-job-user.config file
+  ansible.builtin.template:
+    src: "20-job-user.config.j2"
+    dest: "/etc/condor/config.d/20-job-user.config"
+    mode: "0644"
```

Comment:
These are some additional configuration files to enable partitionable slots and to set up the trusted UID domain.